src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -100,6 +100,21 @@
               </v-btn>
             </div>
           </v-col>
+          <v-col cols="1" v-else-if="isSpecialResolutionFiling">
+            <div class="d-flex justify-end align-end align-sm-start">
+             <v-tooltip
+                    top
+                    content-class="top-tooltip"
+                    transition="fade-transition"
+                    nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>{{ addressChangeInfo }}</span>
+        </v-tooltip>
```

Comment:
I'm leaning towards having this `<v-tooltip>` inside `<div>` in line 92, so that you don't have to specify another class. But there might be some small margin change to the right.

Reply:
I think we can stick with this way as it's more clear. No need to change anything.